controllers/istio_controller.go

```diff
@@ -75,13 +75,13 @@ func NewIstioReconciler(client client.Client, scheme *runtime.Scheme, restConfig
 
 // charts to deploy in the operator namespace (and their suffixes)
 var systemCharts = map[string]string{
-	"istio-cni": "-cni",
+	"cni": "-cni",
 }
 
 // charts to deploy in the istio namespace (and their suffixes)
 var userCharts = map[string]string{
-	"base":                          "-base",
-	"istio-control/istio-discovery": "-istiod",
+	"base":   "-base",
+	"istiod": "-istiod",
```

Comment:
Does this work regardless if the charts are coming from GH or from the released zip files?

Reply:
Yes. The download-charts.sh script transforms the directory structure so that it matches the one in the published charts. See https://github.com/maistra/istio-operator/blob/8150bebec26d2c1b8de0209cc206679760e68dac/hack/download-charts.sh#L54-L58